tests/mpi/distribute_mg_dofs_periodicity_01.mpirun=28.output

```diff
@@ -0,0 +1,7 @@
+
+DEAL::Number of cells: 1024
```

Comment:
The run time on my notebook (2 cores + hyperthreading) is ~60s in debug mode on 28 MPI processes, as compared to ~30s on 6 processes. For a test machine with 10+ core I don't think the difference will be too large.